_layouts/default.html

```diff
@@ -1,12 +1,10 @@
----
-editable: false
----
-
 <!DOCTYPE html>
 <html lang="en">
 {%- include head.html -%}
 
-<body class="{{ page.class }}">
+{% assign body_classes = page.body_classes | join: " " %}
```

Comment:
```suggestion
{% assign body_classes = page.body_classes | join: " " | where: 'body_classes', empty %}
```

🔗 https://michaelcurrin.github.io/dev-cheatsheets/cheatsheets/jekyll/liquid/conditionals/non-empty.html